docs/odh/oci-model-storage.md

```diff
@@ -0,0 +1,208 @@
+# Using OCI containers for model storage
+
+Starting ODH 2.18, the ability to use OCI containers as storage for models
+is enabled in KServe by default. The benefits of using OCI containers for
+model storage are described in [the upstream KServe project
+documentation](https://kserve.github.io/website/latest/modelserving/storage/oci/),
+which also describes how to deploy models from OCI images. 
+
+This page offers a guide similar to the upstream project documentation, but
+focusing on the OpenDataHub and OpenShift characteristics. To demonstrate
+how to create an OCI image, the publicly available [MobileNet v2-7
+model](https://github.com/onnx/models/tree/main/validated/vision/classification/mobilenet)
+is used. This model is in ONNX format.
+
+The ODH projects provides configurations for the OpenVINO model server, which
+supports models in ONNX format. Thus, this guide will use this model server
+to demonstrate how deploy the MobileNet v2-7 model stored in an OCI image.
+
+## Storing a model in an OCI image
+
+Start by creating an empty directory for downloading the model and creating
+the necessary support files to create the OCI image. You may use a temporary
+directory by running the following command:
+```shell
+cd $(mktemp -d)
+```
+
+OpenVINO expects a specific directory tree for model versioning.
+Starting from some base directory, its contents should be a collection of
+numbered subdirectories using positive integer values. The numbers would
+represent the versions of the model. When using OCI images, this
+structure may be irrelevant, as you can use the OCI container registry
+features. However, since OpenVINO expects the versioned directory structure, a
+single subdirectory with an artificial version `1` can be used. Using `models/` as the
+base path, create the expected directory structure and download the sample
+model to it:
+```shell
+# 
+mkdir -p models/1
+
+DOWNLOAD_URL=https://github.com/onnx/models/raw/main/validated/vision/classification/mobilenet/model/mobilenetv2-7.onnx
+curl -L $DOWNLOAD_URL -O --output-dir models/1/
+```
+
+> [!TIP]
+> If you are planning to use a different model server, you should adapt this
+> guide accordingly to your model server requirements. Typically, you would need
+> to place your model files directly under `models/`.
+
+Create a file named `Containerfile` with the following contents:
+```Dockerfile
+FROM registry.access.redhat.com/ubi8/ubi-micro:latest
+COPY --chown=0:0 models /models
```

Comment:
as @spolti mentioned, I don't understand why you want to use root for the directory?

Reply:
Read the reply: https://github.com/opendatahub-io/kserve/pull/415/files#r1788160657

Reply:
I disagree with your point. It is not just about not using root for executing processes; any file or folder included in the pod should have permissions that only allow a user or group to access it. Granting permissions that allow anyone to use those files or folders poses significant security risks. Therefore, I believe we need to change the owner/group of all files/folders we need to use to a user other than root

Reply:
It gives only read permission for group of the specific models and added USER so I think it should be ok for now.